tests/samples/test-two-arrays-with-spaces.json

```diff
@@ -0,0 +1,15 @@
+{"one": 1, \t "two": 2.2}{"one": 10, "two": 22}
+{"one": 1, \n"two": 2.2}{"one": 10, "two": 22}
+{"one": 1, \n\r"two": 2.2}{"one": 10, "two": 22}
+{"one": 1, \n"two": 2.2}\n{"one": 10, "two": 22}
+{"one": 1, \n\r"two": 2.2}\n\r{"one": 10, "two": 22}
+{"one": 1, \n"two": 2.2}\n{"one": 10, "two": 22}\n
+{"one": 1, \n\r"two": 2.2}\n\r{"one": 10, "two": 22}\n\r
+{"one": 1, "two": 2.2}{"one": 10, \n"two": 22}
+{"one": 1, "two": 2.2}{"one": 10, \n\r"two": 22}
+{"one": 1, "two": 2.2}\n{"one": 10, \n"two": 22}
+{"one": 1, "two": 2.2}\n\r{"one": 10, \n\r"two": 22}
+["one", \n "two"]["uno", "dos"]
+["one", \n "two"]\n["uno", "dos"]
+["one", \n\r "two"]["uno", "dos"]
+["one", \n\r "two"]\n\r["uno", "dos"]
```

Comment:
These are not carriage return/tab characters: they're literal backslash and `n`, `r`, `t` in the text file. I meant for these to be independent tests (given as Python strings in the Python test file). If they're in a standalone JSON file like this, it's all one test, rather than testing features like "extra carriage return in the first line" and "extra carriage return in the second line," etc.

Reply:
Thanks! Yes, the strings are added as independent tests and the tests pass. I'll remove the file.

Reply:
done